coverage/clover.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
todo: usually `coverage/` is not committed to the repo because it's a generated doc. We add it to .gitignore and keep it locally only.